### PR TITLE
feat(argo-events): add priorityClassName and podLabels options

### PR DESCRIPTION
**Pull request by t3mi**
Checklist:

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by oliverbaehler** [comment]
@t3mi Thanks for your contribution, Sorry that it took some time for someone to review it. Could you pull the latest master and resolve potential conflicts and bump to a new version? After I can merge it :)

**Comment by t3mi** [comment]
@oliverbaehler done.

File: charts/argo-events/templates/sensor-controller-deployment.yaml [hunk 1]
```diff
@@ -50,6 +53,9 @@ spec:
               port: 8081
             initialDelaySeconds: 3
             periodSeconds: 3
+      {{- if .Values.sensorController.priorityClassName }}
+      priorityClassName: {{ .Values.sensorController.priorityClassName | quote }}
```
Comment (oliverbaehler) [inline charts/argo-events/templates/sensor-controller-deployment.yaml:57]:
```suggestion
      {{- with .Values.sensorController.priorityClassName }}
      priorityClassName: {{ . | quote }}
```

File: charts/argo-events/templates/sensor-controller-deployment.yaml [hunk 2]
```diff
@@ -18,6 +18,9 @@ spec:
       labels:
         app: {{ .Release.Name }}-{{ .Values.sensorController.name }}
         release: {{ .Release.Name }}
+        {{- if .Values.sensorController.podLabels }}
+        {{- tpl (toYaml .Values.sensorController.podLabels) $ | nindent 8 }}
```
Comment (oliverbaehler) [inline charts/argo-events/templates/sensor-controller-deployment.yaml:22]:
```suggestion
        {{- with .Values.sensorController.podLabels }}
        {{- tpl (toYaml .) $ | nindent 8 }}
```

File: charts/argo-events/templates/eventsource-controller-deployment.yaml [hunk 3]
```diff
@@ -50,6 +53,9 @@ spec:
               port: 8081
             initialDelaySeconds: 3
             periodSeconds: 3
+      {{- if .Values.eventsourceController.priorityClassName }}
+      priorityClassName: {{ .Values.eventsourceController.priorityClassName | quote }}
```
Comment (oliverbaehler) [inline charts/argo-events/templates/eventsource-controller-deployment.yaml:57]:
```suggestion
      {{- with .Values.eventsourceController.priorityClassName }}
      priorityClassName: {{ . | quote }}
```

File: charts/argo-events/templates/eventsource-controller-deployment.yaml [hunk 4]
```diff
@@ -18,6 +18,9 @@ spec:
       labels:
         app: {{ .Release.Name }}-{{ .Values.eventsourceController.name }}
         release: {{ .Release.Name }}
+        {{- if .Values.eventsourceController.podLabels }}
+        {{- tpl (toYaml .Values.eventsourceController.podLabels) $ | nindent 8 }}
```
Comment (oliverbaehler) [inline charts/argo-events/templates/eventsource-controller-deployment.yaml:22]:
```suggestion
        {{- with .Values.eventsourceController.podLabels }}
        {{- tpl (toYaml .) $ | nindent 8 }}
```

File: charts/argo-events/templates/eventbus-controller-deployment.yaml [hunk 5]
```diff
@@ -52,6 +55,9 @@ spec:
               port: 8081
             initialDelaySeconds: 3
             periodSeconds: 3
+      {{- if .Values.eventbusController.priorityClassName }}
+      priorityClassName: {{ .Values.eventbusController.priorityClassName | quote }}
```
Comment (oliverbaehler) [inline charts/argo-events/templates/eventbus-controller-deployment.yaml:59]:
```suggestion
      {{- with .Values.eventbusController.priorityClassName }}
      priorityClassName: {{ . | quote }}
```

File: charts/argo-events/templates/eventbus-controller-deployment.yaml [hunk 6]
```diff
@@ -18,6 +18,9 @@ spec:
       labels:
         app: {{ .Release.Name }}-{{ .Values.eventbusController.name }}
         release: {{ .Release.Name }}
+        {{- if .Values.eventbusController.podLabels }}
+        {{- tpl (toYaml .Values.eventbusController.podLabels) $ | nindent 8 }}
```
Comment (oliverbaehler) [inline charts/argo-events/templates/eventbus-controller-deployment.yaml:22]:
```suggestion
        {{- with .Values.eventbusController.podLabels }}
        {{- tpl (toYaml .) $ | nindent 8 }}
```

**Review by oliverbaehler — CHANGES_REQUESTED** [review]
I also recommend using `with`, since you achieve the same affect but don't have to write the same value path twice. If you could accept the suggestions for the sake of consistency :rotating_light:

**Review by oliverbaehler — APPROVED** [review]
lgtm

**Pull request merged**